src/constants/errors.ts

```diff
@@ -1,3 +1,4 @@
-export const WRONG_TRIGGER = "Only pull_request, pull_request_review and pull_request_review_comment triggers are supported";
+export const WRONG_TRIGGER =
+  "Only pull_request, pull_request_review and pull_request_review_comment triggers are supported";
```

Comment:
This diff is from `npm run lint` which happens regardless of this Pull Request's diff.